sphinx/domains/std.py

```diff
@@ -659,14 +615,6 @@ def add_program_option(self, program, name, docname, labelid):
         # type: (str, str, str, str) -> None
         self.data['progoptions'][program, name] = (docname, labelid)
 
-    def check_consistency(self):
```

Comment:
The superclass; `Domain` class still provides empty `check_consistency()` method. So this is not breaking change.